stories/assets/js/FilterAndSearchBar.js

```diff
@@ -0,0 +1,53 @@
+import { Chips } from "../../Atom/Buttons-and-labels/Chips/chips";
+const toggleFilter = function () {
+    $(document).on('click', '.toggleSearch', function () {
+        $('.select-wrapper').toggleClass('show_filter');
+    })
+    var searchOption = $(".options li input");
+    const chipsWrapper = $('.search_filter .Selected_chips');
+
+    $(searchOption).on('click', function () {
+        const el = $(this);
+        const eleId = el.attr('id');
+        if (el.is(':checked')) {
+        const ReactChips=Chips({label:"demo",  variant:'Cross'}).props;
+            const optionValue = $(this).siblings().text();
+            let chips = document.createElement("a");
+            const catText = document.createTextNode(optionValue);
+            chips.appendChild(catText);
+            chips.setAttribute('class',ReactChips.className);
+            chips.setAttribute('href',"javascript:void(0);");
+            chips.setAttribute('option-name', eleId)
+            $(chipsWrapper).append(chips);
+            if($(".select-wrapper").find('.clear_section').length == 0){
+             $(chipsWrapper).after("<div class='clear_section'><span class='clear_chips'>Clear all</span></div>")
+            }
+
```

Comment:
@Ashish-srij Don't use anything specific to React as it can be used with any CMS or framework so this code should be in js or jquery